c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -29,6 +39,7 @@ INSTANCE_ID =
 endif
 
 PIP_CMD ?= $(VENV_BIN)/pip
+PIP_INSTALL_ARGS += install --trusted-host pypi.camptocamp.net
```

Comment:
You can remove the `installed.cfg` file :-)

As I see you don't have any reason to need this `--trusted-host` because the server `pypi.camptocamp.net` shouldn't be used ..., if it steal needed we should see it together :-)